db/schema.rb

```diff
@@ -612,8 +612,6 @@
     t.string "phone"
     t.string "email"
     t.string "password"
-    t.boolean "standard"
```

Comment:
Wondering if we should introduce a new migration for the removal of these columns from the schema? Or was the migration below not run as of yet? https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/engineering/backend/database-migrations.md#schema-updates

Reply:
No, that makes sense. https://github.com/department-of-veterans-affairs/vets-api/pull/6096 makes both columns ignored; after that I'll update this PR to introduce a migration to remove them from the schema.

Reply:
💯